express_relay/sdk/js/src/examples/SimpleSearcher.ts

```diff
@@ -1,9 +1,68 @@
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
-import { checkHex, Client } from "../index";
+import { BidStatusUpdate, checkHex, Client, Opportunity } from "../index";
 import { privateKeyToAccount } from "viem/accounts";
 import { isHex } from "viem";
 
+const DAY_IN_SECONDS = 60 * 60 * 24;
+
+class SimpleSearcher {
+  private client: Client;
+  constructor(
+    public endpoint: string,
+    public chainId: string,
+    public privateKey: string
+  ) {
+    this.client = new Client(
+      { baseUrl: endpoint },
+      undefined,
+      this.opportunityHandler.bind(this),
+      this.bidStatusHandler.bind(this)
+    );
+  }
+
+  async bidStatusHandler(bidStatus: BidStatusUpdate) {
+    console.log(`Bid status for bid ${bidStatus.id}: ${bidStatus.status}`);
+  }
+
+  async opportunityHandler(opportunity: Opportunity) {
+    const bid = BigInt(argv.bid);
```

Comment:
this was useful internally as a dummy test to make sure auction server ignores lower bids. I'd run two scripts with different bid params and see who wins. I removed it from the docs, but will leave it here since there is some default value and it will not complicate the users testing flow.